### ISSUE TITLE
Accessability Mode

### ISSUE THREAD

**Issue by TormundsMember** [TormundsMember/Easy_Flashcards#42]
**Is your feature request related to a problem? Please describe.**
Currently the app is not optimized at all. Screens don't work for accessibility, some buttons and menu-items are not labeled for talkback at all.

**Describe the solution you'd like**
Apply the basic accessibility implementation for all screens.

**Describe alternatives you've considered**
An additional mode that exists next to the none-accessible mode and that is focused on accessibility.
